src/js/controller/instream-adapter.js

```diff
@@ -201,7 +201,7 @@ var InstreamAdapter = function(_controller, _model, _view, _mediaPool) {
         if (skipoffset) {
             // Start background loading once the skip button is clickable
             _this.setupSkipButton(skipoffset, _options);
-            _backgroundLoadPosition = skipoffset;
+            _backgroundLoadPosition = typeof skipoffset === 'string' ? utils.seconds(skipoffset) : skipoffset;
```

Comment:
- `utils.seconds(skipoffset)` has a type check do you should just use it, don't add a check around it.
- skipOffset can be a percentage in which case this won't work - look at asktipbutton.js and consider updating and reading adModel 'skipOffset' after the time has been updated
- When skipOffset is a percent it will not be calculated until the ad duration is known so this should be async